tests/queries/0_stateless/01174_select_insert_isolation.sh

```diff
@@ -39,11 +39,12 @@ function thread_select()
         # Result of `uniq | wc -l` must be 1 if the first and the last queries got the same result
```

Comment:
This comment should now be something like
"The first and the last queries must get the same result"
Btw, why was the `uniq | wc -l` approach flaky?